### PR TITLE
RATEST-157: Adding page functionality for manage user test workflow

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-157)
Description -> Adding page functionality for manage user account test workflow

### CODE REVIEW

**Comment by HerbertYiga** [comment]
looking good to me cc @ibacher @brandones

**Comment by ibacher** [comment]
@kdaud Do we have any tests exercising this new page functionality?

File: ui-tests/src/test/java/org/openmrs/reference/page/ManageUserAccountPage.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        clickOn(GENDER_FEMALE);
+    }
+
+    public void clickOnAddUserAccount() {
+        clickOn(ADD_USER_ACCOUNT);
+    }
+   
+    public void enterUsername(String username) {
+        findElement(USER_NAME).clear();
+        findElement(USER_NAME).sendKeys(username);
```
Comment (ibacher) [inline ui-tests/src/test/java/org/openmrs/reference/page/ManageUserAccountPage.java:70]: I'm of two minds about this pattern. On the one hand, its necessary to reliably do what the test needs to do. On the other hand, if there's data pre-populated into that field, that seems like an issue that should be flagged. Thoughts?

**Comment by kdaud** [comment]
> Do we have any tests exercising this new page functionality?

The test [here](https://issues.openmrs.org/browse/RATEST-157) is relying on the page functionality in this commit.

File: ui-tests/src/test/java/org/openmrs/reference/page/ManageUserAccountPage.java (hunk 1 above)
Comment (kdaud) [reply ui-tests/src/test/java/org/openmrs/reference/page/ManageUserAccountPage.java:70]: Here is my thought on the epic:
Reliably making the pattern to directly do what the logic is supposed to perform would be ok, however ,on the the other hand there could be data pre-populated in the field and invoking  clear() method makes things right using the same pattern. 
For-example when editing user account details the same pattern will cater for such scenarios than again writing another pattern that can edit the data, so I thought catching two birds in one net would be cool than using two nets to catch two birds. 

Echoing on the first thought invoking clear() method has no harm at all but does the magic for the second  thought!

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
Thanks @kdaud. This LGTM.

**Pull request merged**